src/mappings/mappingHandlers.ts

```diff
@@ -171,6 +173,35 @@ export async function handleExecuteContractMessage(msg: CosmosMessage<ExecuteCon
   await msgEntity.save();
 }
 
+export async function handleCW20Transfer(msg: CosmosMessage<CW20TransferMsg>): Promise<void> {
+  logger.info(`[handleCW20Transfer] (tx ${msg.tx.hash}): indexing CW20Transfer ${messageId(msg)}`)
+
+  const id = messageId(msg);
+  const {
+    sender,
+    contract,
+    msg: {
+      transfer: {
+        recipient,
+        amount
+      }
+    }
+  } = msg.msg.decodedMsg;
```

Comment:
Maybe we need to sneak some extra checks here to filter out messages having a "transfer" method call, but not actually from a CW20 contract. See https://github.com/fetchai/ledger-subquery/blob/main/src/mappings/mappingHandlers.ts#L228-L233

Reply:
Thanks Flavien! Added some checks.